static/app/stores/committerStore.tsx

```diff
@@ -13,7 +13,7 @@ type State = {
   };
 };
 
-type CommitterStoreInterface = {
+interface CommitterStoreDefinition extends Reflux.StoreDefinition {
```

Comment:
since we replaced `type` with `interface`, wasn't it better to have an interface suffix?

Reply:
Nah, it's redundant to redeclare the kind of type this is in the name as well. Also that is prone to just not being accurate over time as types change but namings stay the same, best to just avoid it